.travis.yml

```diff
@@ -2,15 +2,17 @@ os:
   - linux
   - osx
   - windows
-services:
-  - redis-server
 language: node_js
 node_js:
   - '6'
   - '8'
   - '10'
   - '12'
 before_install:
+  - 'if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository ppa:chris-lea/redis-server -y; fi'
```

Comment:
Geht glaube ich auch etwas schöner: https://docs.travis-ci.com/user/installing-dependencies/#installing-packages-with-the-apt-addon

Reply:
I exactly got that soluton from the same page more up https://docs.travis-ci.com/user/installing-dependencies/#installing-packages-from-a-custom-apt-repository

It was not working else when I remember